packages/core-web/src/styles/markbind.css

```diff
@@ -103,6 +103,128 @@ pre > code.hljs[heading] {
         width: 3em;
         height: 100%;
     }
+
+    /* CSS from codeblock-light.min.css */
```

Comment:
Rather than copying the CSS, it might be better to add the class name to the code block? What do you think